cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -12,13 +12,69 @@ class Teams_Common {
   static #teamsOverviewNavigationButton = '[data-testid="Teams"]'
   static #messangerActivation = '[data-testid="rocketchat_checkbox"]'
   static #saveButton = '[data-testid="create_team_btn"]'
+  static #newsTabOnTeamDetail = '[data-tab="js-news"]'
+  static #createNewsButtonOnTeamDetail = '[data-testid="create_a_team"]'
+  static #threeDotManageTeam = '[id="page-title"]'
+  static #manageTeamMembersOption = '[data-testid="administrate_team_members"]'
+  static #addInternalTeamMemberButton = '[data-testid="internal_team_members"]'
+  static #selectInternalTeamMember = '[data-testid="select_team_members_add"]'
+  static #studentTableBody = '[data-testid="students_names_container"]'
+  static #confirmTeamMemberAddButton = '[data-testid="btn-submit"]'
+  static #confirmTeamMemberDeleteButton= '[data-testid="btn-submit"]'
+  static #deleteIconInTableViewRow = '[data-testid="btn-delete-team-member"]'
+
+
+  doNotSeeDeletedStudentInTeam(){
+    cy.get(Teams_Common.#studentTableBody).find('tr td','Herbert').should('not.exist')
+  }
+
+  removeStudentInTeam () {
+    cy.get(Teams_Common.#studentTableBody).contains('tr', 'Herbert').then(tableRow => {
+      cy.wrap(tableRow).find(Teams_Common.#deleteIconInTableViewRow).click()
+      cy.get(Teams_Common.#confirmTeamMemberDeleteButton).contains('Teilnehmer:in löschen').click()
```

Comment:
Instead of the german button content text a data-testid, which identifies the button, should be used.
About the data-testid see comment in line 23